### ISSUE TITLE
list tutorial

### ISSUE THREAD

**Issue by neighborhood999** [EragonJ/Kaku-website#25]
List tutorial and parse title to `json` file：

```
{
  'how-to-back-up-music': {
    'en': 'How to back up music',
    'zh-TW': '如何備份音樂'
  },
  ...etc
}
```

This has useful for Kaku app catch tutorial and make Q&A page.

**Comment by neighborhood999** [comment]
#24 already add it.

**Comment by EragonJ** [comment]
@neighborhood999 lol when did you add this, i didn't notice that !! sorry for the late reply because I had been sick for few days 😷

**Comment by neighborhood999** [comment]
@EragonJ It's ok！What？Are you OK？Hope you get well soon.

**Comment by EragonJ** [comment]
fixed at #24
